mod_reforged/hooks/skills/actives/miasma_skill.nut

```diff
@@ -0,0 +1,27 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/miasma_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = "Summon a toxic miasma in the target area dealing damage to anything that lives and breathes!";
+	}
+
+	// Vanilla doesn't have a getTooltip function defined for this skill
+	q.getTooltip = @() function()
+	{
+		local ret = this.skill.getDefaultUtilityTooltip();
+		ret.push({
+			id = 10,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = ::Reforged.Mod.Tooltips.parseString("The targeted tiles gain miasma that lasts " + ::MSU.Text.colorGreen(3) + " [rounds|Concept.Round]")
+		});
+		ret.push({
+			id = 11,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = ::Reforged.Mod.Tooltips.parseString("All characters except the undead receive " + ::MSU.Text.colorRed(5) + " - " + ::MSU.Text.colorRed(10) + " damage when they end their [turn|Concept.Turn] in the miasma")
+		});
```

Comment:
This skill is missing any mention of the fact it replaces any other tile effect. See for example `throw_fire_bomb_skill` or [my version of the miasma skill description](mod_reforged/hooks/skills/actives/miasma_skill.nut). I prefer mine as it is much more general.

Reply:
I'm trying to avoid making the skill tooltips "too mechanical" because that doesn't fit within the theme of this game. I think mentioning that it replaces any existing tile effect is not crucial to this skill and secondly may become invalid if some mod etc. or even Reforged later allows the framework of having multiple tile effects per tile.

We can remove the mention of the damage if it's too specific. I wasn't too happy about adding it myself either because that also suffers from being too rigid and prone to being incorrect if something is changed in miasma damage function.

Reply:
Please refer to `throw_fire_bomb_skill` or smoke bomb. It is currently vanilla convention to mention it in the tooltip that an effect replaces existing tile effects.
Either remove it everywhere or add it everywhere.
I am in favor of keeping this line in (like vanilla does it), but improving the wording.